### ISSUE TITLE
clicks bombs

### ISSUE THREAD

**Issue by mohammady1403** [Wokzy/blum-bot#35]
Thanks of blum-bot. It seem new version of xmas is not efficient as much as previous version to leave alone the bombs.

**Comment by belloabdul0** [comment]
Hi Mohammad, would you be kind enough to show me how I can set this up on my PC, I've installed python...

**Comment by HamedDN** [comment]
yeah man thats clicking the bombs too i am getting at most 100 per a ticket

**Comment by kirube** [comment]
the updated one is not efficiency, do you guys knows any other bot like this?
